components/browser/state/src/main/java/mozilla/components/browser/state/state/BrowserState.kt

```diff
@@ -28,6 +28,6 @@ data class BrowserState(
     val containers: Map<String, ContainerState> = emptyMap(),
     val extensions: Map<String, WebExtensionState> = emptyMap(),
     val media: MediaState = MediaState(),
-    val queuedDownloads: Map<Long, DownloadState> = emptyMap(),
-    val search: SearchState = SearchState()
+    val search: SearchState = SearchState(),
+    val downloads: Map<Long, DownloadState> = emptyMap()
```

Comment:
Let's not alter the order of the parameter as this could cause breakage for consumers that are not using [Named Arguments](https://kotlinlang.org/docs/reference/functions.html#named-arguments). `downloads` was before of`search`.